models/Sauce.js

```diff
@@ -0,0 +1,19 @@
+// Importation de Mongoose
+const mongoose = require("mongoose");
+
+// Création d'un schéma de données (informations requises)
+const sauceSchema = mongoose.Schema({
+  name: { type: String, required: true },
+  manufacturer: { type: String, required: true },
+  description: { type: String, required: true },
+  mainPepper: { type: String, required: true },
+  imageUrl: { type: String, required: true },
+  heat: { type: Number, required: true },
+  likes: { type: Number, required: true },
+  dislikes: { type: Number, required: true },
+  usersLiked: { type: ["String <userId>"], required: true },
```

Comment:
["String <userId>"] not exist, it's juste represent the fact that you hav to link with user model.

For example :  usersLiked: [{ type: String, ref: "User" }],